test/pid_parameters_tests.cpp

```diff
@@ -57,8 +57,9 @@ void check_set_parameters(
   const double I_MAX = 10.0;
   const double I_MIN = -10.0;
   const bool ANTIWINDUP = true;
+  const bool SAVE_ITERM = false;
```

Comment:
I would propose going with `true` because it is not default value. So we are sure everything is set.